addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -305,3 +305,7 @@ $o-wblog-loader-size: 50px;
         background-color: rgba(black, 0.7);
     }
 }
+
+#o_wblog_index_content .container .row .border-end {
+    width:auto
+}
```

Comment:
- I would put `>` between everything to be even more precise.
- Is it not possible to ensure that we have the layout list and the sidebar?
- Also a space and a semicolon are missing in the CSS rule ;)

Reply:
> Is it not possible to ensure that we have the layout list and the sidebar?

Unfortunately, there is no indication in the DOM that we are in "List" layout or "Grid" layout. Indeed, at the toggle between "Grid" layout and "List" layout, there is a rpc call to `/website/theme_customize_data` that will activate a view corresponding to the "List" layout. However, there is no class added in the DOM indicating if we are in "List" layout or "Grid" layout.
However, the presence of a `<div class="border-end"></div>` inside a `.row` element already ensure that we are in "List" layout and that the sidebar is displayed. Indeed, if we are in "Grid" layout and the sidebar is shown or if we are in "List" layout and the sidebar is hidden, then there will be no `<div class="border-end"></div>` inside a `.row` element.